### PR TITLE
fix: Fix message input not being disabled when form message is not submitted

### PR DESCRIPTION
**Pull request by liamcho**
Fixes [AC-3597](https://sendbird.atlassian.net/browse/AC-3597)

[AC-3597]: https://sendbird.atlassian.net/browse/AC-3597?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/GroupChannel/components/MessageInputWrapper/index.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export interface MessageInputWrapperProps {
 
 export const MessageInputWrapper = (props: MessageInputWrapperProps) => {
   const context = useGroupChannelContext();
-  return <MessageInputWrapperView {...props} {...context} />;
+  return <MessageInputWrapperView {...props} {...context}/>;
```
Comment (bang9) [inline src/modules/GroupChannel/components/MessageInputWrapper/index.tsx:17]: revert please!

Comment (liamcho) [reply src/modules/GroupChannel/components/MessageInputWrapper/index.tsx]: thanks, it's been reverted.

File: src/modules/GroupChannel/components/MessageInputWrapper/MessageInputWrapperView.tsx [hunk 2]
```diff
@@ -22,8 +23,6 @@ import { useLocalization } from '../../../../lib/LocalizationContext';
 import SuggestedMentionList from '../SuggestedMentionList';
 import { useDirtyGetMentions } from '../../../Message/hooks/useDirtyGetMentions';
 import { SendableMessageType } from '../../../../utils';
-// import QuoteMessageInput from '../../../../ui/QuoteMessageInput';
-// import VoiceMessageInputWrapper from './VoiceMessageInputWrapper';
```
Comment (bang9) [inline src/modules/GroupChannel/components/MessageInputWrapper/MessageInputWrapperView.tsx:26]: 이거 두줄도 복구좀 부탁드립니다.

**Review by bang9 — APPROVED** [review]

File: src/modules/GroupChannel/components/MessageInputWrapper/MessageInputWrapperView.tsx (hunk 2 above)
Comment (liamcho) [reply src/modules/GroupChannel/components/MessageInputWrapper/MessageInputWrapperView.tsx]: 넵 돌려놓았습니다

**Pull request merged**